package.json

```diff
@@ -12,57 +12,57 @@
     "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
   },
   "dependencies": {
-    "@babel/compat-data": "^7.8.6",
-    "@mdi/font": "^5.0.45",
-    "@sentry/browser": "^5.15.4",
-    "@sentry/integrations": "^5.15.4",
+    "@babel/compat-data": "^7.11.0",
```

Comment:
These are the versions actually used (per the lock file).
Updating these has no impact.